superset/assets/cypress_build.sh

```diff
@@ -29,6 +29,7 @@ flask run -p 8081 --with-threads --reload --debugger &
 
 #block on the longer running javascript process
 time npm ci
+time npm run install-cypress
```

Comment:
This step seems to be redundant as the script is installing `cypress` library which is already coming via package.json dependencies.